py_ecc/bls/hash.py

```diff
@@ -1,7 +1,14 @@
-import math
-import hashlib
 import hmac
-from typing import Union
+import math
+from typing import (
+    Union,
+)
+from hashlib import sha256
+from _hashlib import HASH  # type: ignore
```

Comment:
@hwwhww, I'd appreciate advice on typing here. `mypy` complains if this is not ignored and `flake8` doesn't like it if it is. I'm trying not to add additional mypy errors so #89 can be merged and mypy added to CI.

Reply:
You can make both of them ignore this by commenting like

```python
from _hashlib import HASH  # type: ignore  # noqa: <flake8-error-code>
```

Where `<flake8-error-code>` is something like `E501` (typically included in the error output from flake8

Reply:
@CarlBeek sorry I missed you ping! 🙏
It seems fine to remove the comments: https://github.com/hwwhww/py_ecc/commit/39d8878471f987b4322dc209b56faf0b2043b0b2

I guess you were using different versions of flake8/mypy from the CI?